files/scripts/supervisor-proc-exit-listener

```diff
@@ -1,17 +1,34 @@
 #!/usr/bin/env python
 
+import getopt
 import os
 import signal
 import sys
 import syslog
 
+import swsssdk
+
 from supervisor import childutils
 
 # Contents of file should be the names of critical processes (as defined in
 # supervisor.conf file), one per line
 CRITICAL_PROCESSES_FILE = '/etc/supervisor/critical_processes'
 
-def main():
+# This table in databse contains the features for container and each
+# feature for a row will be configured a state or number.
+CONTAINER_FEATURE_TABLE_NAME = 'CONTAINER_FEATURE'
+
+def main(argv):
+    container_name = None
+    opts, args = getopt.getopt(argv, "c:", ["container-name="])
+    for opt, arg in opts:
+        if opt in ("-c", "--container-name"):
+            container_name = arg
+
+    if not container_name:
+        syslog.syslog(syslog.LOG_ERROR, "Container name not specified. Exiting...")
+        sys.exit(2)
```

Comment:
I only wanted you to change the other exit code to 2 (line 62). Please keep this exit code as 1 so they can be differentiated.

Reply:
Misunderstood. Updated.